code/modules/mob/living/carbon/human/human.dm

```diff
@@ -256,6 +257,14 @@
 				stat("Internal Atmosphere Info", internal.name)
 				stat("Tank Pressure", internal.air_contents.return_pressure())
 				stat("Distribution Pressure", internal.distribute_pressure)
+
+		if(back && istype(back, /obj/item/weapon/rig))

```

Comment:
Checking for `back` should be redundant, considering istype is going to return false if `back` is null.